src/vendorcode/fsp/coffeelake/build.rs

```diff
@@ -1,143 +0,0 @@
-extern crate bindgen;
```

Comment:
Why is this being removed?

Reply:
It'll only build after running `git submodule update --init --recursive`. Perhaps we should add this to the CI.

Reply:
Reverted this change and added it back to the list of crates that do not work with clippy yet.